server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -221,9 +234,10 @@ impl RequisitionTransferTester {
         response_store: &StoreRow,
         item1: &ItemRow,
         item2: &ItemRow,
+        thread_number: u32,
     ) -> RequisitionTransferTester {
         let request_requisition = inline_init(|r: &mut RequisitionRow| {
-            r.id = uuid();
+            r.id = format!("{}_request_requisition", thread_number);
```

Comment:
This helps with the debug output and looking at the db, but could go back to UUID here?